Dynatrace/dynatrace-operator
# Removes the oneagent immutable image feature-flag

# Description

The customised oneagent image can handle the additional args/files provided by the operator when immutable image support is enabled. 
This means the feature flag is redundant and the behaviour can just default to immutable image support.


## How can this be tested?
Buy deploy not adding any feature flag the following is true
- `<dynakube-name>-oneagent-tenant-secret` is created in the operator's namespace
- "--set-tenant" "--set-server" args are provided the oneAgent daemonset
- "connection-info-secret" volume and volumeMount is present on the oneAgent daemonset
	- should mount the `<dynakube-name>-oneagent-tenant-secret` secret

## Checklist
